### DISCUSSION TITLE
syslog-ng CRL check when CRL is expired

### DISCUSSION THREAD

**Discussion by BRKiranAT** [syslog-ng/syslog-ng#5199] [Q&A]
I need some clarification regarding the following issue. 
In the TLS configuration, if the CRL has expired but the peer certificate is still valid, the data is being blocked between the syslog-ng client and server. 
I believe that the CRL should be ignored in this case, allowing the data to be encrypted and sent to the peer. 
Can someone help me understand this better?

**Comment by bazsi** [comment]
CRLs are only checked at initial negotiation/key exchange and not checked during while the connection is open.
